File: LeetCode/2535. Difference Between Element Sum and Digit Sum of an Array/solution.cpp
/**
 * Runtime: 8 ms
 * Memory Usage: 15.2 MB 
 */
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int a = 0, b = 0;
        for (auto num: nums) {
            a += num;
            while (num > 0) {
                b += num % 10;
                num /= 10;
            }
        }
        return abs(a - b);
    }
};

File: LeetCode/2535. Difference Between Element Sum and Digit Sum of an Array/solution.py
"""
Runtime: 56 ms
Memory Usage: 15.2 MB
"""
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        a = b = 0
        for num in nums:
            a += num
            while num > 0:
                b += num % 10
                num //= 10
        return abs(a - b)
    